spec/models/user_spec.rb

```diff
@@ -1226,4 +1226,36 @@
       end
     end
   end
+
+  describe '#onboarding - feature toggle enabled' do
+    let(:user) { create(:user, icn: identity_icn) }
+    let(:identity_icn) { 'some_identity_icn' }
+
+    before do
+        Flipper.enable(:veteran_onboarding_beta_flow, user)
+    end
+
+    it 'show_onboarding_flow_on_login returns true when feature flag is enabled and display_onboarding_flow is true' do
```

Comment:
Totally optional, but you could wrap some of this in context blocks. Probably separate by when flag is enabled vs disabled. This will make the it blocks shorter and more readable and group your specs more cohesively.

Reply:
I think with that change, your describe block could just be '#onboarding'.

Reply:
I like this idea! Will fix